TypeInjections/Tool/Tool.fs

```diff
@@ -21,31 +21,38 @@ module Tool =
         let commitRepo = newRepo.Lookup<Commit>(hash)
         Commands.Checkout(newRepo, commitRepo) |> ignore
 
-    // based off command line flags, create and provide path for input/output folders
-    let interpretFlag (argvList: string list) =
-        let flag = argvList.Item(0)
-
-        // -p [inputPath] [inputPath] [outputPath]
-        if flag.Contains("-p") then
-            Console.WriteLine("***** local project comparison")
-
-            if argvList.Length <> 4 then
-                failwith "usage: program -p OLD NEW OUTPUTFOLDER"
 
-        // -l [git commit hash] [git commit hash] [OutputPath]
-        elif flag.Contains("-l") then
-            Console.WriteLine("***** git commit")
+    type options =
+        { [<Option('d', "delete", Required = false, HelpText = "use option to remove New and Old folders")>]
+          delete: bool
+          [<Option('l', "local", Required = false, HelpText = "[inputPath] [inputPath] [outputPath]")>]
+          local: seq<string>
+          [<Option('g', "git", Required = false, HelpText = "[old commit] [new commit] [outputPath]")>]
+          git: seq<string>
+          [<Option('r', "remote", Required = false, HelpText = "[github repo url] [new commit] [old commit] [OutputPath]")>]
```

Comment:
We mix multiple words, lower camel case and upper camel case here, please be consistent

Reply:
What's the difference between `remote` and `git`?

Reply:
Will fix the capitalization issue. 
`Remote` takes in a github repository from the internet (the repo is not cloned onto the local computer). `git` is when you are in a folder with a `.git` file and want to compare versions of that repository

Reply:
Why do we need to support `remote`? Isn't `git` enough?

Reply:
Hira and I thought it would be a convenient feature to have